TombEngine/Objects/TR3/Entity/TonyBoss.cpp

```diff
@@ -91,28 +104,27 @@ namespace TEN::Entities::Creatures::TR3
 			fx->pos.Orientation.y = flame.yRot;
 			fx->pos.Orientation.z = 0;
 			fx->objectNumber = ID_TONY_BOSS_FLAME;
+			fx->flag1 = (short)flame.type;
 			fx->speed = flame.speed;
 			fx->color = Vector4::Zero;
-			fx->flag1 = flame.type;
-			fx->flag2 = (GetRandomControl() & 3) + 1;
-
 			switch (flame.type)
 			{
-			case T_ZAPPDEBRIS:
+			case TonyFlameType::ZAppDebris:
```

Comment:
What does `ZApp` even mean?

Reply:
Fixed and renamed to what they really mean.